src/main/java/core/db/DBUpdater.java

```diff
@@ -295,19 +298,18 @@ private void updateDBv500(int dbVersion, int version) throws SQLException {
 				trainingTable.tryDeleteColumn("YEAR");
 				trainingTable.tryDeleteColumn("WEEK");
 
+				// Fill training table using information from VEREIN, PLAYER and XTRADATA (Step 2)
+				var sql = "select ACTIVATIONDATE FROM BASICS LIMIT 1";
+				rs = m_clJDBCAdapter.executeQuery(sql);
+				if (rs != null) {
+					rs.next();
+					var activationdate = rs.getTimestamp("ACTIVATIONDATE").toInstant();
+					// TODO all happens on construction. I would prefer a static method for such jobs
```

Comment:
this will only create a training week manager but then we need to call a function to store the vector inside TrainingTable

Reply:
OK - thought that would be done implicitly